Makefile

```diff
@@ -41,6 +41,10 @@ DOCKER_USER=$(shell id -u):$(shell id -g)
 all: install-tools markdownlint markdown-link-check misspell table-check compatibility-check schema-check \
 		 check-file-and-folder-names-in-docs
 
+.PHONY: all-skip-link-check
+all: install-tools markdownlint misspell table-check compatibility-check schema-check \
```

Comment:
This should be updated to all-skip-link-check

Reply:
Fixed when I renamed the target in [8ccd778](https://github.com/open-telemetry/semantic-conventions/pull/1586/commits/8ccd778a05529eca1bd3598e220ccff5f1cfd558)